Sources/Networking/Caching/CustomerInfoCallback.swift

```diff
@@ -15,7 +15,7 @@ import Foundation
 
 struct CustomerInfoCallback: CacheKeyProviding {
 
-    typealias Completion = (Result<CustomerInfo, BackendError>) -> Void
+    typealias Completion = @Sendable (Result<CustomerInfo, BackendError>) -> Void
```

Comment:
this is what the tests are complaining about, right? 
I don't know how to solve it cleanly, though - it looks like Xcode 14 doesn't like the idea of mixing concurrency stuff with non-concurrency

Reply:
Removed!